### ISSUE TITLE
Script returns 0 byte files

### ISSUE THREAD

**Issue by aidenmitchell** [SynArchive/flyover-re-fixed#1]
Hi there, it looks like this doesn't work anymore, unfortunately. I ran the example (`./run.sh 34.007603 -118.499741`) and got 3 zero byte files. If I had to guess, Apple's APIs aren't returning the proper data anymore, but I don't develop in Go.

**Comment by schaale** [comment]
Ditto

**Comment by felixgeen** [comment]
sadly, same here. I think apple maps changed something with the C3M data

**Comment by SynArchive** [comment]
@aidenmitchell Yeah, it's not Go's fault, I was able to even port the whole thing to python while it was still working, the APIs still function, as if you don't pass it the right auth parameters it fails, but if you provide the right like "checksum", they just return empty files, quite frustrating haha.. This was just something to fix the old repo. If someone finds something I'll update the repo and publish the python port

**Comment by mpalenque** [comment]
any news??
